src/main/kotlin/org/codefreak/codefreak/entity/Evaluation.kt

```diff
@@ -24,7 +25,7 @@ class Evaluation(
   @ColumnDefault("1970-01-01 00:00:00")
   var evaluationSettingsFrom: Instant
 ) : BaseEntity() {
-  @OneToMany(mappedBy = "evaluation", cascade = [CascadeType.ALL], orphanRemoval = true)
+  @OneToMany(mappedBy = "evaluation", cascade = [CascadeType.ALL], orphanRemoval = true, fetch = FetchType.EAGER)
```

Comment:
I think we should not force erager fetching here because it may not be needed in all cases and this is non-reversible. Instead, create a query for explicitly enable eager fetching. See als https://vladmihalcea.com/eager-fetching-is-a-code-smell/